src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/TempPE/DesignTimeInputs.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
```

Comment:
Missing copyright header in this and other files.

Reply:
Thanks, I _always_ forget this. I wish there was an analyzer for it.

Reply:
`StyleCop.Analyzers` offers this in [SA1636](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1636.md).

Reply:
Yeah, CPS has that on, but my experience with StyleCop Analyzers does not enamour me to turn them on.

Reply:
It's slow in that solution because there are so many hidden rules and thousands of diagnostics. If that was all cleaned up, I don't think you'd notice a perf impact. It's also no fun with overly prescriptive rules.